commit 93afdf12ebbac253044439a5ed2247e1732de48e (refs/changes/98/1098/2)
Author: Cody Peter Mello <cody.mello@joyent.com>
Date:   2016-12-09T00:21:29+00:00 (2 years, 10 months ago)
    
    FWAPI-269 Describe how owner_uuid scopes rules in fwadm(1M) introduction
    Reviewed by: Robert Mustacchi <rm@joyent.com>
    Approved by: Robert Mustacchi <rm@joyent.com>

diff --git a/src/fw/man/fwadm.1m.md b/src/fw/man/fwadm.1m.md
index 22fe3d1d..ac1eeef6 100644
--- a/src/fw/man/fwadm.1m.md
+++ b/src/fw/man/fwadm.1m.md
@@ -43,8 +43,10 @@ The format is similar to the vmadm(1M) format with most properties omitted
 and some simplified properties. See the "REMOTE VMS", "REMOTE VM PROPERTIES"
 and "EXAMPLES" sections below for details.
 
-Firewall rules only apply to VMs that have the firewall_enabled property set
-to true. Adding, updating or deleting firewall rules or remote VMs will reload
+Firewall rules only apply to VMs that have the firewall\_enabled property set
+to true. Rules with an owner\_uuid are scoped to VMs with a matching owner.
+Global rules, which are ownerless, will apply to all VMs described in their
+targets. Adding, updating or deleting firewall rules or remote VMs will reload
 the firewalls of any VMs affected.
 
 
diff --git a/src/fw/man/fwadm.1m.md.in b/src/fw/man/fwadm.1m.md.in
index 8b45c5d5..ee2c07ec 100644
--- a/src/fw/man/fwadm.1m.md.in
+++ b/src/fw/man/fwadm.1m.md.in
@@ -43,8 +43,10 @@ The format is similar to the vmadm(1M) format with most properties omitted
 and some simplified properties. See the "REMOTE VMS", "REMOTE VM PROPERTIES"
 and "EXAMPLES" sections below for details.
 
-Firewall rules only apply to VMs that have the firewall_enabled property set
-to true. Adding, updating or deleting firewall rules or remote VMs will reload
+Firewall rules only apply to VMs that have the firewall\_enabled property set
+to true. Rules with an owner\_uuid are scoped to VMs with a matching owner.
+Global rules, which are ownerless, will apply to all VMs described in their
+targets. Adding, updating or deleting firewall rules or remote VMs will reload
 the firewalls of any VMs affected.
 
 
